opencga-catalog/src/main/java/org/opencb/opencga/catalog/db/mongodb/iterators/OrganizationCatalogMongoDBIterator.java

```diff
@@ -105,7 +107,50 @@ private void fetchNextBatch() {
                 }
             }
 
+            // TODO: Remove this code after several releases (TASK-5923)
+            recoverSecretKeyTask5923(organizationDocument);
+
             organizationListBuffer.add(organizationDocument);
         }
     }
+
+    // TODO: Remove this method after several releases (TASK-5923)
+    /**
+     * This method should only be necessary in order to be able to run the migration for TASK-5923 because OpenCGA looks for the config.
+     * This configuration was changed in TASK-5923, therefore, OpenCGA would be unable to initialise without it.
+     * Once the migration is run, this code will do nothing, so it should be removed in the future.
+     *
+     * @param organizationDocument Organization document.
+     */
+    private void recoverSecretKeyTask5923(Document organizationDocument) {
```

Comment:
Once this method is deleted, the corresponding migration should be also "somehow disabled", so if it's still pending, it fails and forces you to jump to an earlier version